aracne/hpAracne.java

```diff
@@ -15,39 +15,37 @@
 import java.util.Locale;
 import java.util.Random;
 
-import aracne.BootstrapConsolidator;
-import aracne.DPI;
-import aracne.MI;
-
-import common.DataParser;
-import common.ExpressionMatrix;
 import common.DataMatrix;
+import common.DataParser;
 import common.DataVector;
+import common.ExpressionMatrix;
 
 
-public class hpAracne {
+public class hpAracne{
 	// Variable definition
 	static NumberFormat formatter = new DecimalFormat("0.###E0");
 	static float simCut = 0; // In the original paper this was a parameter. E.g. if two TFs have a very high MI, DPI is not calculated
-	final static int geneNumberInMIthresholding = 1000000;
+	final static int geneNumberInMIthresholding = 1000;

```

Comment:
I'm a bit confused about this parameter. In ARACNe-AP, it is set to 3000, whereas hpARACNe suggests a value of 1000000. Does this parameter need to be adjusted for different applications? Should we make it a user-accessible parameter?